test/index.test.ts

```diff
@@ -7,6 +7,7 @@ import { Application, Probot } from "../src";
 import { ProbotOctokit } from "../src/octokit/probot-octokit";
 
 import path = require("path");
+import { version } from "../package.json";
```

Comment:
no need to import package.json here, the version is always `0.0.0-development`. It's replaced before publishing to npm via [semantic-release](https://github.com/semantic-release/semantic-release)

```suggestion
```

Reply:
Got it. Can I force push a new commit with the fix?

Reply:
no need to force push. Don't worry about the commit history in the pull request, I'll squash & merge it into a single commit anyway